singer-io/tap-jira
# TDL-19174 skipped the record for out of range date values

# Description of change
- Skipped the record, when SchemaMismatch is thrown due to date out of range value error

# Manual QA steps
 - Verify that the record is skipped for out of range date values
 - Verify that the record is successfully written for valid date values
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
